shift4/tokens.py

```diff
@@ -2,8 +2,8 @@
 
 
 class Tokens(Resource):
-    def create(self, params):
-        return self._post("/tokens", params)
+    def create(self, params, request_options=None):
```

Comment:
/tokens endpoint doesn't have the idempotency key feature

Reply:
fixed.